packages/themes/src/g10.js

```diff
@@ -196,7 +196,7 @@ export const buttonSecondaryActive = activeSecondary;
 export const buttonTertiaryActive = activeTertiary;
 
 export const focusInset = inverse01;
-export const focuseInverse = inverseFocusUi;
+export const focusInverse = inverseFocusUi;
```

Comment:
@jnm2377 just a heads up, I noticed we had an extra `e` in here so I've updated it in this PR 👍🏻